torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -187,6 +187,26 @@ bool conv2dIsSupported(const torch::jit::Node* node) {
   return true;
 }
 
+// The fuser currently only supports matmul of 2D x 2D matrices
+bool matmulIsSupported(const torch::jit::Node* node) {
```

Comment:
Is there a reason for putting this in kernel instead of tensorexpr_fuser ?

Reply:
Only because we already have `conv2dIsSupported` there. I wouldn't mind moving it to `tensorexpr_fuser.cpp`, but in this PR for consistency I'd prefer to keep them in `kernel.cpp`